tests/Traits/ValidCodeAnalysisTestTrait.php

```diff
@@ -76,6 +79,10 @@ public function testValidCode(
         $codebase->enterServerMode();
         $codebase->config->visitPreloadedStubFiles($codebase);
 
+        foreach ($codebase_config as $key => $value) {
```

Comment:
This seems to just be adding more issues than it may solve. I'm wondering if I should just hardcode `max_shaped_array_size` to a lower value for all tests?

Reply:
ok, let's keep it the way it was and just have a really long test